src/slice.rs

```diff
@@ -9,10 +9,11 @@ use crate::dimension::slices_intersect;
 use crate::error::{ErrorKind, ShapeError};
 use crate::{ArrayViewMut, DimAdd, Dimension, Ix0, Ix1, Ix2, Ix3, Ix4, Ix5, Ix6, IxDyn};
 use alloc::vec::Vec;
-use std::convert::TryFrom;
```

Comment:
Are these changes to the imports necessary? When the `std` feature is not enabled, `core` is aliased to the name `std`.

Reply:
OK, I restore `use std::*` changes.